sourcegraph/sourcegraph
# Fix Cody ScopeSelector UI

- Turn multi-colored embeddings status indicator to grey.
- Fix dark mode bg color on hover for list item in scope selector.
- Add the clear search button back as `type="search"` does not work on Firefox.

## Test plan

- visit /cody
- turn on dark mode
- open scope selector 
- hover over repo, bg color should be dark
- enter in search box, there should be a close icon on the right.
